bin/carmi.spec.js

```diff
@@ -0,0 +1,27 @@
+const pify = require("pify");
+const { exec } = require("child_process");
+const path = require("path");
+const asyncExec = pify(exec);
+const BINARY_PATH = path.resolve(__dirname, "carmi");
+
+const runBinary = args => asyncExec(`${BINARY_PATH} ${args}`);
+
+describe("carmi binary", () => {
+  it("has a help menu", async () => {
+    const helpMessage = await runBinary("--help");
+    expect(helpMessage).toMatch(/shows this very help message/);
```

Comment:
can also be

```suggestion
    expect(helpMessage).toMatchSnapshot();
```

for storing the help message. But I don't think it's relevant right now